pkg/controllers/dynakube/apimonitoring/reconciler.go

```diff
@@ -19,8 +20,13 @@ func NewReconciler(dtc dtclient.Client, clusterLabel, kubeSystemUUID string) *Re
 	}
 }
 
-func (r *Reconciler) Reconcile() error {
-	objectID, err := r.createObjectIdIfNotExists()
+const (
+	settingsSchemaId      = "builtin:cloud.kubernetes"
+	appTransitionSchemaId = "builtin:app-transition.kubernetes"
```

Comment:
do we want them to be duplicated? https://github.com/Dynatrace/dynatrace-operator/pull/2281/files#diff-2d6b21fa20ddac7f3c9773d874e19e38ed9706d5aafb9c0e01a7aee578e459b3R85

Reply:
really good point, how can I reference them from another file ?  actually I was trying to do it, but then I have thought maybe different package is an issue?

Reply:
export them by starting them with a capital letter

Reply:
thanks for explanations, issue is fixed